web/react/stores/channel_store.jsx

```diff
@@ -270,4 +270,5 @@ ChannelStore.dispatchToken = AppDispatcher.register(function(payload) {
     }
 });
 
+ChannelStore.setMaxListeners(11);
```

Comment:
I needed to up the max allowed listeners so I could put a channel store listener on the notifications tab